src/doublylinkedlist.js

```diff
@@ -0,0 +1,158 @@
+'use strict'
+
+class Node {
+
+  constructor( data ) {
+    this.data = data
+    this.next = null
+    this.prev = null
+  }
+
+}
+export default class DoublyLinkedList {
+
+  constructor () {
+      this.head = null
+      this.tail = null
+      this.count = 0
+
+
+  }
+  // Inserts a node (with the provided data) to the head of the list
+  insertFirst ( value ) {
+    //create new node
+    const node = new Node( value )
+    //find if head is defined
+    if ( this.head ) {
+      //set old head to next of new node
+      node.next = this.head
+      this.head.prev = node
+      //if head is defined, set new node to head
+    } else {
```

Comment:
We don't need an else statement here, we can just place the "this.tail = node" right on top of the "this.head" declaration.